customer-managed/aws/terraform/iam_utility_node_group.tf

```diff
@@ -558,6 +558,37 @@ data "aws_iam_policy_document" "load_balancer_controller_2" {
       }
     }
   }
+
+  dynamic "statement" {
+    for_each = var.enable_private_link ? ["true"] : []
+    content {
+      effect = "Allow"
+      actions = [
+        "elasticloadbalancing:RegisterTargets",
+        "elasticloadbalancing:DeregisterTargets",
+      ]
+      resources = [
+        "arn:aws:elasticloadbalancing:${var.region}:${local.aws_account_id}:targetgroup/*-rp-*",
+        "arn:aws:elasticloadbalancing:${var.region}:${local.aws_account_id}:targetgroup/*-kf-*/*",
```

Comment:
The seed target groups are not included here. The names should end with `-seed`. Wonder how the test passed.  I will create a cluster and check the names of PL target groups.

Reply:
I looked at the target group names that are created and it seemed like the seed ones were all `-rp-seed` or `-kf-seed` and were therefore covered by these rules.